src/components/organisms/Header/types.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface HeaderProps {
```

Comment:
`SEOProps`, `LayoutProps` and `HeaderProps` are the same. Maybe it would be good to add this interface in one place and then extend the others with it? It would be also good to have the `SEO` component added in one place like `Layout` or `Header` so that we won't have to import it on every page.